compiler/rustc_codegen_llvm/src/intrinsic.rs

```diff
@@ -1628,7 +1628,7 @@ unsupported {} from `{}` with element `{}` of size `{}` to `{}`"#,
             out_elem
         );
     }
-    macro_rules! arith {
+    macro_rules! arith_bin {
```

Comment:
These just get defined once and invoked once, so they don't need to be abbreviated. How about `arith_binary` and `arith_unary`?

Reply:
Thanks for pointing. It has been corrected.